cpuid.go

```diff
@@ -231,13 +231,27 @@ var flagNamesArm = map[ArmFlags]string{
 	GPA:      "GPA",      // Generic Pointer Authentication
 }
 
+// x86 Advanced Matrix Extensions features, in CPUInfo.AmxFeatures
+const (
+	AMXBF16 = 1 << iota // Tile computational operations on BFLOAT16 numbers
```

Comment:
Lets make these easy to distinguish.
```suggestion
	AMXBF16 AmxFlags = 1 << iota // Tile computational operations on BFLOAT16 numbers
```

Reply:
Thanks, that's much better! I was a bit worried because `go` compiled `CPUInfo.Features&AMXBF16` without a warning. After this change it gives `mismatched flags` error.

Reply:
(Fixed in updated commit with one more related change.)